frontend/packages/volto-light-theme/src/theme/sticky-menu.scss

```
@@ -34,24 +35,24 @@ body.cms-ui .sticky-menu {
       font-size: 10px;
       font-style: normal;
       font-weight: 700;
-      line-height: 18px; /* 180% */
+      line-height: 12px;
       text-transform: capitalize;
     }
 
     li {
       display: flex;
-      height: 93px;
+      height: 97px;
```

```suggestion
      min-height: 95px;
```

I'd recommend using `min-height: 95px` to allow items to grow with multi-line text and prevent content cutoff. If the design requires fixed height, we should ask the designer for an example showing the intended behavior when text wraps to multiple lines.